### ISSUE TITLE
[Bug Report] Missing german translation

### ISSUE THREAD

**Issue by Talmit** [Pyrdacor/Ambermoon-Advanced#28]
**Describe the bug**
When reading the sign for crystal's wine cellar, the german translation is missing.

**To Reproduce**
1. Start the game using the german language
2. Go to 446 / 195
3. Read the sign
4. The description is in english

**Expected behavior**
The description should be german

**Screenshots**
![image](https://github.com/Pyrdacor/Ambermoon-Advanced/assets/88923192/80063eef-2c80-4502-9dec-1d917443618c)

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version 1.9.4 / Ambermoon Advanced 1.3

**Additional context**
The attached Save 3 has the appropriate setup. 
[Saves.zip](https://github.com/Pyrdacor/Ambermoon-Advanced/files/13831899/Saves.zip)

**Comment by Pyrdacor** [comment]
Yeah this is a known bug. I fixed this a long time ago in the Ambermoon Advanced game data but I never updated the AA version until then. Maybe I can add this little fix somehow.

**Comment by Talmit** [comment]
I don't think it's that important, I just noticed it and figured I'd mention it. It's not as though it's much of an issue during the playthrough :)

**Comment by Pyrdacor** [comment]
Finally fixed in 1.30 (available in remake, Amiga version is coming soon).
